modules/trusted-firmware-m/Kconfig

```diff
@@ -96,6 +125,9 @@ config TFM_ISOLATION_LEVEL
 	int "Isolation level setting."
 	range 1 3
 	depends on BUILD_WITH_TFM
+	default 1 if TFM_PROFILE_TYPE_SMALL
```

Comment:
Note that these do not update if TFM_PROFILE_TYPE is updated via menuconfig.

Reply:
Yes, but we should not care, right? I made the option hidden when the profile symbol is set, anyways, to reflect that the TFM will configure its own isolation level and the symbols choice, here, won't matter.